CIP-????/README.md

```diff
@@ -0,0 +1,97 @@
+---
+CPS: ????
```

Comment:
The containing folder is currently `CIP-????` so can you please rename it to `CPS-`something... preferably something unique e.g. `data-structures` since this is from your `master` branch & you might want to submit another one?

Reply:
um, I very much picked the wrong branch here, oops :facepalm: I guess I'll leave it for now :(

Reply:
@michaelpj no problem here for now... but if another CIP comes in on the `master` branch it will have to use a different subdirectory name 😅